pkg/assembler/backends/ent/schema/certifyvex.go

```diff
@@ -62,7 +62,9 @@ func (CertifyVex) Edges() []ent.Edge {
 // Indexes of the VEX.
 func (CertifyVex) Indexes() []ent.Index {
 	return []ent.Index{
-		index.Fields("known_since", "justification", "status", "statement", "status_notes", "origin", "collector", "document_ref").Edges("vulnerability", "package").Unique().Annotations(entsql.IndexWhere("artifact_id IS NULL")),
-		index.Fields("known_since", "justification", "status", "statement", "status_notes", "origin", "collector", "document_ref").Edges("vulnerability", "artifact").Unique().Annotations(entsql.IndexWhere("package_id IS NULL")),
+		index.Fields("known_since", "justification", "status", "statement", "status_notes", "origin", "collector", "document_ref").
+			Edges("vulnerability", "package").Unique().Annotations(entsql.IndexWhere("artifact_id IS NULL")).StorageKey("vex_artifact_id"),
```

Comment:
`StorageKey` was needed here (and the other locations) as the atlas was getting confused and thought it was the same index twice.